spec/lib/fetch_book_data_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+require 'vcr_helper'
+
+describe FetchBookData, type: :lib, vcr: VCR_OPTS do
+  describe '#subjects' do
+    subject { described_class.new.subjects }
+
+    context 'success' do
+      let(:book_subjects) do
+        ["Math", "Science", "Humanities", "Social Sciences", "Business", "Essentials", "College Success", "High School"]
+      end
+
+      example do
+        expect(subject).to match(book_subjects)
```

Comment:
when the CMS changes, and adds a subject. this test would start failing.  is that intentional?

Reply:
Would it? I think it won't because we use the VCR gem, so the responses are cached (until you update them when you choose to by re-recording the request/response).

Reply:
i mean this when VCR is cleared.

Reply:
In that case then, yes. I think so. Or what could/should I do instead?